pkg/pmem-csi-driver/controllerserver-node.go

```diff
@@ -31,7 +31,7 @@ type nodeVolume struct {
 	Params map[string]string `json:"parameters"`
 }
 
-func (nv *nodeVolume) Copy() *nodeVolume {
+func (nv nodeVolume) Copy() *nodeVolume {
```

Comment:
Isn't Go now forced to create a copy of the `nodeVolume` instance prior to calling `Copy`? Why not keep the function as it was?

Note that I am not worried about the performance overhead. Go even recommends to pass parameters by value instead of pointer and optimizes for that. I just don't understand the purpose of this change.

Reply:
Yes, you might be true. The change might break the intention, and unnecessary. The reason behind the change was previously we had a pointer to nodeVolume while calling Copy(), whereas now we have the value of it.